src/OrchardCore.Modules/OrchardCore.Lucene/GraphQL/LuceneQueryFieldTypeProvider.cs

```diff
@@ -39,23 +42,35 @@ public async Task<IChangeToken> BuildAsync(ISchema schema)
                 var name = query.Name;
                 var source = query.Source;
 
-                var querySchema = JObject.Parse(query.Schema);
-
-                var type = querySchema["type"].ToString();
-
-                if (query.ReturnContentItems &&
-                    type.StartsWith("ContentItem/", StringComparison.OrdinalIgnoreCase))
+                try
                 {
-                    var contentType = type.Remove(0, 12);
-                    var queryField = BuildContentTypeFieldType(schema, contentType, query);
-                    if (queryField != null)
+                    var querySchema = JObject.Parse(query.Schema);
+                    if (!querySchema.ContainsKey("type"))
```

Comment:
We have to add a sample Schema on the page ,but I don't know the right schema for here.

Reply:
https://github.com/OrchardCMS/OrchardCore/issues/6554

Reply:
Please leave that for a seperate pr